Cargo.toml

```diff
@@ -5,7 +5,8 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1.0"
-as-ffi-bindings = { git = "https://github.com/massalabs/as-ffi-bindings.git", tag = "v0.5.3" }
+# as-ffi-bindings = { git = "https://github.com/massalabs/as-ffi-bindings.git", tag = "v0.5.3" }
+as-ffi-bindings = { path = "../as-ffi-bindings_2" }
```

Comment:
Once as-ffi-bindings's PR merged, tag and update the tag here. Otherwise approved